ArduPlane/quadplane.cpp

```diff
@@ -637,8 +634,7 @@ bool QuadPlane::setup(void)
       that the objects don't affect the vehicle unless enabled and
       also saves memory when not in use
      */
-    motor_class = (enum AP_Motors::motor_frame_class)frame_class.get();
-    switch (motor_class) {
+    switch ((AP_Motors::motor_frame_class)frame_class) {
```

Comment:
```suggestion
    switch (frame_class) {
```

no cast required, kind of the point of `AP_Enum<>` :-)